app/MainHie.hs

```diff
@@ -120,15 +126,15 @@ run opts = do
   d <- getCurrentDirectory
   logm $ "Current directory:" ++ d
 
-  let vomitOptions = defaultOptions { boLogging = BlVomit}
-  let defaultOpts = if optGhcModVomit opts then vomitOptions else defaultOptions
+  let initOpts = defaultCradleOpts { cradleOptsVerbosity = verbosity }
+      verbosity = if optBiosVerbose opts then Verbose else Silent
       -- Running HIE on projects with -Werror breaks most of the features since all warnings
       -- will be treated with the same severity of type errors. In order to offer a more useful
       -- experience, we make sure warnings are always reported as warnings by setting -Wwarn
-      biosOptions = defaultOpts { boGhcUserOptions = ["-Wwarn"] }
+--      ghcModOptions = defaultOpts { GM.optGhcUserOptions = ["-Wwarn"] }
```

Comment:
`-Wwarn` is now set with the other options. This comment can be deleted.

Reply:
Also done in https://github.com/mpickering/haskell-ide-engine/pull/14